src/index.js

```diff
@@ -47,8 +49,22 @@ app.get('/:query', async (req, res) => {
   const inviteCode = await InviteResolver.resolve(req.params.query)
   logger.info(`Rendering ${inviteCode}`, { label: 'Renderer' })
   const inviteSVG = await InviteRenderer.render(inviteCode, req.query)
-  res.setHeader('Content-Type', 'image/svg+xml')
-  res.send(inviteSVG)
+
+  switch (req.query.format.toLowerCase()) {
```

Comment:
This will cause the request to fail if format is not passed

Reply:
One could fix the other issue as well (don't recall the #) to handle failed requests gracefully

Reply:
Ok, I've just realized the fuck up, don't mind me. Will fix it soon.